Announcement for ProtoSchool's first libp2p tutorial

terichadbourne opened the pull request:
WIP

Do not merge until we've done these things (hopefully Wed/Thu of this week): 
- [x] Publish the tutorial (in final reviews at https://github.com/ProtoSchool/protoschool.github.io/pull/574) 
- [x] Blog proofs by @zebateira & @yusefnapora 
- [x] Replacement of header image (change Multiformats logo to libp2p)
- [x] Improve layout of GIFs - formatting was copied over from ProtoSchool and will likely need to be updated in this context

zebateira commented:
Blog Post PR here: https://github.com/ipfs/blog/pull/517

zebateira commented on content/post/124-protoschool-libp2p.md:
```diff
@@ -0,0 +1,39 @@
+    <img class="w-75-ns w-100" src="/124-libp2p-comes-to-protoschool/libp2p-ipfs-animation.gif" alt="libp2p animation representing libp2p being extracted out of ipfs" />
+</div>
+
+Historically, companies that produced peer-to-peer applications like like Skype or BitTorrent created their own protocols to support them. Those protocols made a lot of assumptions about the environments they'd run in and the needs they'd meet, making them incredibly hard to upgrade or adapt to new environments. libp2p, however, has been extracted from its original implementation as the networkin ...
```
```suggestion
Historically, companies that produced peer-to-peer applications like like Skype or BitTorrent created their own protocols to support them. Those protocols made a lot of assumptions about the environments they'd run in and the needs they'd meet, making them incredibly hard to upgrade or adapt to new environments. libp2p, however, has been extracted from its original implementation as the networking stack of IPFS, and can now serve a broad variety of use cases. While everyone building on IPFS is using libp2p as a dependency, many folks are using libp2p independently or embedded in other projects such as X, Y, and Z (insert examples that don't use IPFS).
```

Since Filecoin uses IPFS, maybe it's not necessary to mention it + IPFS is more of a dev tool than Filecoin I believe, so just using IPFS sounds better.
Just 2 cents.

terichadbourne replied:
@zebateira I'm inclined to leave this because (a) not everyone knows Filecoin uses IPFS and (b) libp2p sounds like it's more popular if we include both. @yusefnapora What do you think? Could either of you help with these examples I didn't insert? 😂

zebateira reviewed (changes requested):
LGTM 💅 
Just one details that I'm not sure about about IPFS v Filecoin and two typos.

jennwrites commented:
Projects that use libp2p: [Vac](https://vac.dev/), [Matrix](https://twitter.com/matrixdotorg/status/1264002081498845186?s=20), [Golem](https://golem.network/), [Polkadot](https://polkadot.network/), just to name a few.

terichadbourne commented:
> Projects that use libp2p: [Vac](https://vac.dev/), [Matrix](https://twitter.com/matrixdotorg/status/1264002081498845186?s=20), [Golem](https://golem.network/), [Polkadot](https://polkadot.network/), just to name a few.

Thanks @jennwrites! Do you know that those ones used libp2p independent of IPFS, which is what I'm going for, or shall I check in with the libp2p crew to confirm?

terichadbourne commented:
The deploy is now in progress for the tutorial itself, so I hope to be able to publish this post later this afternoon once we've confirmed it's live. I've updated to today's date and fixed my header image, GIFs, and examples, so I think we're good to go as soon as that's resolved.

jennwrites commented:
re: timing sounds great! 

The libp2p examples I'm fairly certain are outside of IPFS, but please feel free to confirm!
